src/event.js

```diff
@@ -104,17 +104,26 @@ jQuery.event = {
 
 	add: function( elem, types, handler, data, selector ) {
 
```

Comment:
all of the code looks good do me

Reply:
@22aam stop spamming or you’ll get blocked